### DISCUSSION TITLE
SampleRTT

### DISCUSSION THREAD

**Discussion by ChaofengWu0** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#53] [Q&A]
请问sampleRTT 在没有设置timeout的时候 每次都要发一个包查询一下sampleRTT吗
不能一次查询一直使用是吗

**Comment by TANSixu** [comment]
hi，sample rtt可以通过每次发包和收包来获得。比如 记录发包时间和收包时间 或者 在header加入时间戳

**Comment by TANSixu** [comment]
hi，sample rtt可以通过每次发包和收包来获得。比如 记录发包时间和收包时间 或者 在header加入时间戳
